barberia/src/App/pages/TimePicker.js

```diff
@@ -23,12 +23,13 @@ const AppoPicker = (props) => {
   React.useEffect(() => {
     const onTimeSelect = event => setTime(event.time);
     let picker = new AppointmentPicker(pickerRef.current, options);
```

Comment:
Why are you using `let` on `picker` and `currentRef`? These variables are not being mutated, use `const` instead.

Reply:
fixed